src/test/java/com/google/api/codegen/testdata/discoveries/go/go_storage.v1.json.baseline

```diff
@@ -1474,6 +1474,8 @@ func main() {
     // TODO: Add desired fields of the request body.
   }
 
+  // TODO: Use `Download` in place of `Do` below to download media content. For more information, see:
+  // https://godoc.org/google.golang.org/api/storage/v1
```

Comment:
IMO it's a good idea to do it - GoDoc has a predictable format based on the names of types, and so it should always link correctly as long as we're up to date.